schemas/groups/electrical.json

```diff
@@ -174,6 +178,16 @@
             }
           }]
         }
+      },
+      "chargeTransferred": {
+        "type": "number",
+        "description": "Amount of charge produced(+) or consumed(-) since last report",
```

Comment:
These are akin to trip meters, but I don't like manual (user initiated) resets; just will not get done on many installs. Some charging devices could auto-reset: alternators could start at 0 each time they have power applied, solar often resets accumulators at night. But not all are so simple. Wind – at times go on for days? AC charger held in Float mode indefinitely? And batteries..

My thinking for universal applicable is to reset each time an accumulated value is pushed up to the SignalK server. It does not matter if the reporting period is short or long. Time being a component of `J` and `C` no information will be lost, only smaller/larger accumulated values will be pushed up at each report. With the app aware of when a new value is available it could maintain its own a running sum(s) over the duration(s) it wishes to cover.